README.md

```diff
@@ -98,6 +98,19 @@ The pipe's capacity for stdout/stderr can be modified by specifying the `agent.c
 to the guest kernel command line. For example, `agent.container_pipe_size=2097152` will set the stdout and stderr
 pipes to 2097152 bytes.
 
+## Uevent Netlink Socket Receive Buffer Size
+
+When hotplugging a huge size memory into the Kata VM, the kernel in the VM will produce a lot of memory object add
+uevents and send all these uevents to Kata agent by netlink socket. However, default netlink socket receive buffer
```

Comment:
CI is not happy, you can replace the word netlink with \`netlink\` or add that word to `check-spelling` https://github.com/kata-containers/tests/tree/master/cmd/check-spelling/data

```
WARNING: Word 'Netlink': did you mean one of the following?: Netlify
WARNING: Word 'netlink': did you mean one of the following?: Netlify
WARNING: Word 'netlink': did you mean one of the following?: Netlify
WARNING: Word 'netlink': did you mean one of the following?: net link, net-link, interlink, jetliner, nestling, neckline, nightlong, Nataline
WARNING: Word 'netlink': did you mean one of the following?: net link, net-link, interlink, jetliner, nestling, neckline, nightlong, Nataline
ERROR: Spell check failed for file: 'README.md'
INFO: spell check failed for document README.md
ERROR: spell check failed, See https://github.com/kata-containers/documentation/blob/master/Documentation-
```

Reply:
@devimc Hahaha~I think check-spelling scripts should change WARNING to Error level, I am confused with CI output a long time. Thanks for your detailed explanation.